File: blog-dao/src/main/java/com/dao/ContentMapper.java
package com.dao;

import com.entity.Content;

public interface ContentMapper {
    int deleteByPrimaryKey(Integer conid);

    int insert(Content record);

    int insertSelective(Content record);

    Content selectByPrimaryKey(Integer conid);

    int updateByPrimaryKeySelective(Content record);

    int updateByPrimaryKeyWithBLOBs(Content record);
}

File: blog-dao/src/main/java/com/dao/ContentMapper.kt
package com.dao

import com.entity.Content

interface ContentMapper {
    fun deleteByPrimaryKey(conid: Int?): Int

    fun insert(record: Content): Int

    fun insertSelective(record: Content): Int

    fun selectByPrimaryKey(conid: Int?): Content

    fun updateByPrimaryKeySelective(record: Content): Int

    fun updateByPrimaryKeyWithBLOBs(record: Content): Int
}